app/posts/manage-teacher-training-applications/2021-03-18-self-service-registration.md

```diff
@@ -0,0 +1,434 @@
+---
+title: Self-service registration for higher education institutions and their school direct partners
```

Comment:
Can we avoid ‘self service’ as a term?

Reply:
Can you recommend an alternative to 'self-service'?